doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -105,83 +96,47 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       the same key as the left- or right-hand side of the lemma given to rewrite, and the arguments
       are then unified up to full reduction.
 
-.. tacn:: replace @term with @term’
-   :name: replace
+.. tacn:: replace @one_term__from with @one_term__to {? @occurrences } {? by @ltac_expr3 }
 
-   This tactic applies to any goal. It replaces all free occurrences of :n:`@term`
-   in the current goal with :n:`@term’` and generates an equality :n:`@term = @term’`
+   Replaces all free occurrences of :n:`@one_term__from`
+   in the current goal with :n:`@one_term__to’` and generates an equality
+   :n:`@one_term__from = @one_term__to’`
```

Comment:
It was reported in #13480, that this documentation is actually incorrect. The generated equality is reversed.